stories/atoms/components/icon/icon.stories.mdx

```diff
@@ -198,20 +182,42 @@ In particular, this makes it possible to better manage high-contrast elements an
     >
       <div className="icon-card">
         <Icon name="wallet" />
-        <Typography as="span" color="text" fontSize="x-small" fontFamily="secondary">
+        <Typography color="text" fontSize="x-small" fontFamily="secondary">
           Current theme
         </Typography>
       </div>
       <div className="icon-card inverted">
         <Icon name="wallet" invertColor />
-        <Typography as="span" color="inverted-text" fontSize="x-small" fontFamily="secondary">
+        <Typography color="inverted-text" fontSize="x-small" fontFamily="secondary">
           Inverse theme
         </Typography>
       </div>
     </div>
   </Story>
 </Canvas>
 
+## Class Name
+
+The `className` property allows you to style the icon with css properties.  
```

Comment:
```suggestion
The `className` property allows you to style the icon with CSS properties.  
```
I would have capitalized CSS because it is an acronym

Reply:
You are right 👍